engine/src/main/scala/cromwell/engine/workflow/lifecycle/finalization/CopyWorkflowMetadataActor.scala

```diff
@@ -0,0 +1,123 @@
+package cromwell.engine.workflow.lifecycle.finalization
+
+import akka.actor.{ActorRef, LoggingFSM, PoisonPill, Props}
+import cromwell.backend.AllBackendInitializationData
+import cromwell.backend.BackendWorkflowFinalizationActor.{FinalizationFailed, FinalizationSuccess, Finalize}
+import cromwell.core.Dispatcher.{IoDispatcher, ServiceDispatcher}
+import cromwell.core.WorkflowOptions._
+import cromwell.core._
+import cromwell.core.io.AsyncIoActorClient
+import cromwell.core.path.{BetterFileMethods, PathFactory}
+import cromwell.engine.EngineWorkflowDescriptor
+import cromwell.engine.workflow.lifecycle.finalization.CopyWorkflowMetadataActor.{CopyWorkflowMetadataActorData, CopyWorkflowMetadataActorState, GetMetadata, HasReceivedEventsData, IdleData, Initial, WaitingState}
+import cromwell.filesystems.gcs.batch.GcsBatchCommandBuilder
+import cromwell.services.MetadataServicesStore
+import cromwell.services.metadata.MetadataQuery
+import cromwell.services.metadata.MetadataService.{MetadataLookupResponseWithRequester, MetadataServiceKeyLookupFailed, SwitchToWaitMetadata}
+import cromwell.services.metadata.impl.{MetadataDatabaseAccess, ReadDatabaseMetadataWorkerActor}
+import spray.json.JsObject
+import cromwell.services.metadata.impl.builder.MetadataBuilderActor
+import cromwell.services.metadata.impl.builder.MetadataBuilderActor.{BuiltMetadataResponse, ReadyToBuildResponse}
+
+import scala.concurrent.Future
+import scala.concurrent.duration.{Duration, FiniteDuration}
+import scala.util.{Failure, Success}
+
+object CopyWorkflowMetadataActor {
+  def props(workflowId: WorkflowId, ioActor: ActorRef, serviceRegistryActor: ActorRef, workflowDescriptor: EngineWorkflowDescriptor, workflowOutputs: CallOutputs,
```

Comment:
Please consider splitting such long lines. Most of the styleguides are saying that maximum line length should be around `80` characters. For example, you could refer to [this one](https://docs.scala-lang.org/style/)